python/tk_desktop/desktop_window.py

```diff
@@ -92,8 +93,8 @@ def __init__(self, parent=None):
         self.ui.apps_button.setProperty("active", True)
         self.ui.apps_button.style().unpolish(self.ui.apps_button)
         self.ui.apps_button.style().polish(self.ui.apps_button)
-        login = ShotgunLogin.get_instance_for_namespace("tk-desktop")
-        connection = login.get_connection()
+
+        connection = shotgun.get_sg_connection()
```

Comment:
ideally, it would be nice if this could be `sgtk.util.get_shotgun_connection()` and if we documented it in the offcial API docs (here: https://toolkit.shotgunsoftware.com/entries/23808858#Utility%20Functions ).